terminal.py

```diff
@@ -119,12 +119,28 @@ def main():
             main_cmd = True
 
         elif ns_known_args.opt == "candle":
-            candle(
-                s_ticker, (datetime.now() - timedelta(days=180)).strftime("%Y-%m-%d")
-            )
+
+            if s_ticker:
+                candle(
+                    s_ticker,
+                    (datetime.now() - timedelta(days=180)).strftime("%Y-%m-%d"),
+                )
+
+            else:
+                print(
+                    "No ticker selected. Use 'load ticker' to load the ticker you want to look at."
+                )
+                main_cmd = True
```

Comment:
I think this 
`main_cmd = True`
should be outside the else condition.

Reply:
I agree, and fixed. I was just unsure since it wasn't there at all. Didn't want to mess up something i didn't understand :)

Reply:
Yea, I saw it wasn't there. Well spotted! Thank you!

If you scroll a bit down you can see that when you call one of the main menus, they return this b_quit flag which in case is true it exists the program, in case it's false it just means the user wants to go one level up. Since the user will be coming to this main level again, after being in a sub-level, we want to print the main menu again. Hence why if b_quit flag is false we print the help menu.

That main_cmd exists for the case we use one of those main functions, to say that we don't want to show the help menu in those cases, unless the user selects "help".

Hope this made sense! 
Once the checks are done I'll merge this, cheers :)

Reply:
I understood the function, but didn't understand why it wasn't present for candle already, make sense it was just a typo. 

I like this project and ill try to dive into the code and see if i can help in some way :) Keep up the good work